### PR TITLE
JavaScript Capstone:

### PR DESCRIPTION
**Pull request by HabibPayenda**
### **The following requirements were added to the project:**

1. Setup the project by creating a new directory.
2. Set up the repository on GitHub and use Gitflow.
3. Set up webpack.
4. Set up a JavaScript testing library (Jest).
5. Build the home, comments and about interfaces.
6. Used the PokeApi to retrieve the pokemon information
7. Used the Involvement api to retrieve the likes and comments data
8. Implemented jest to the project
9. Link to the project documentation:
    https://drive.google.com/file/d/10I0pO2RM-4va_xkXsOHVjsCcxpBz7KAV/view?usp=sharing

**Please check if we have missed something.**

Thank you

### CODE REVIEW

File: src/modules/addLikes.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  };
+
+  await fetch(url, data);
+  getLikes();
+};
+
```
Comment (Reem-lab) [inline src/modules/addLikes.js:18]: - [ ] i think there is something wrong in the like button so when I press on it the page refresh but I can't see increaseing but when I press again I can see the like so please make it when I press on like it increases without also resreshing the page from the Frist time so kindly work on this error 🚀💯😊✅

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻

Good job so far! 🔥🔥💯💯💯🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Great design and good documentation 🔥🔥🔥
📌 Good commit messages ✅
📌 Professional README ✅


## Required Changes ♻️
_Check the comments under the review._
- [ ] I suggest adding the `MIT.md` file to your project :ok_hand:, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it :wink:
>
> > The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. :100:
>


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addLikes.js (hunk 1 above)
Comment (HabibPayenda) [reply src/modules/addLikes.js:18]: @Reem-lab Thanks for the review, We will work on it and fix the issue.

**Review by Lordkaito — APPROVED** [review]
Hi @David-Kasilia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**